### ISSUE TITLE
sprays bug

### ISSUE THREAD

**Issue by LILITH44444** [ValveSoftware/Dota2-Gameplay#15862]
### Cosmetic Item Name

Tundra Esports fan content , Gaimin gladiators fan content and Talon fan content.

### Description

i bought these sprays earlier it was perfectly fine but now i can't use them, please fix it. (it used to show a pictures of them but now it's just blank grey pixel!) before i bought them it said it will be permanently use .

### Example Match ID

none

### Screenshots

![20231217110928_1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/162727032/1b09423f-9a73-4bcb-a171-4e81e13396e3)
![20240308233728_1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/162727032/dd07dea8-9130-43e5-9ee1-f9ae3230532d)
![20240308233723_1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/162727032/f86a9152-559c-4624-89ec-11d89d00e882)

**Comment by ErikHumphrey** [comment]
Yeah, seems to apply to all Fan Content sprays as of the Act IV update. Here's a video of an Aurora Gaming one not working (applies to all three):

https://github.com/user-attachments/assets/5056535a-b22d-4464-9df3-187086ed9f7e
